spec/rails_helper.rb

```diff
@@ -14,25 +14,11 @@
   config.filter_rails_from_backtrace!
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = true
-end
-
-Capybara.register_driver :headless_chrome do |app|
-  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: { args: %w[headless disable-gpu] }
-  )
 
-  Capybara::Selenium::Driver.new(
-    app,
-    browser: :chrome,
-    desired_capabilities: capabilities
-  )
+  config.before(:each, type: :system) do
+    Capybara.server = :puma, { Silent: true }
```

Comment:
This line suppresses the capybara puma output in test logs.

Before:

```
jon@colossus:~/code/monolithium(switch-to-system-tests)% rspec

Randomized with seed 56887
Capybara starting Puma...
* Version 3.11.4 , codename: Love Song
* Min threads: 0, max threads: 4
* Listening on tcp://127.0.0.1:50263
......................................

Finished in 6.49 seconds (files took 3.83 seconds to load)
38 examples, 0 failures

Randomized with seed 56887
```

After:

```
jon@colossus:~/code/monolithium(switch-to-system-tests)% rspec

Randomized with seed 39631
......................................

Finished in 8.01 seconds (files took 4.52 seconds to load)
38 examples, 0 failures

Randomized with seed 39631
```

Found this solution from here:

https://github.com/rspec/rspec-rails/issues/1897